VL.Data/Interfaces/IBookService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using VL.Shared.Model;
+
+namespace VL.Shared.Interfaces
+{
+    public interface IBookService
+    {
+        public Task<List<Book>> GetBooksAsync();
+
+        public Task<Book?> GetBookAsync(int id);
```

Comment:
Does this return a nullable Book?

Reply:
![image](https://user-images.githubusercontent.com/99216796/195894769-d11e5553-f6da-4dd6-94ff-141fde1a649a.png)

Reply:
It should not, but adding the "?" inside of the return tag stops the possible null ref from displaying. Is there a better way to do this?

Reply:
No, I suppose a Book could not exist - just different syntax versions is all and different if <Nullable> is enabled/disabled project wide